index.bs

```diff
@@ -4422,8 +4452,9 @@ closed</dfn> steps given |window|, |accepted| and optional |user text|
 
         browsingContext.UserPromptOpenedParameters = {
           context: browsingContext.BrowsingContext,
-          type: "alert" / "confirm" / "prompt" / "beforeunload",
+          handler: "accept" / "dismiss" / "ignore",
```

Comment:
Should we also inform the client about the `notify` flag?

Reply:
Again, I don't think it makes sense. For a HTTP session you'll get an error. For a BiDi-only session you are being notified by the event.